litmus-portal/frontend/src/views/CreateWorkflow/ChooseAWorkflowAgent/index.tsx

```diff
@@ -71,6 +71,10 @@ const ChooseWorkflowAgent = forwardRef((_, ref) => {
               },
               workflowIcon: '',
             });
+            workflow.setWorkflowManifest({
+              manifest: '',
+              engineYAML: '',
+            });
```

Comment:
We would do this at the end of the workflow step, at Schedule summary after applying the workflow. That is because all the manifest data will be cleared there itself after saving it up in the previous steps, with your coded approach the manifest will still be there in redux until the user opens/schedules a new workflow again. 

Thus if the user schedules one workflow and goes to template or settings the manifest will be there in redux (Better to clear it after applying itself)

Reply:
Sure, will make the change! Thanks for the review 💯